src/sipmanalyze/estimate.py

```diff
@@ -0,0 +1,121 @@
+"""
+
+estimate.py
+
+Functions to help with estimation of low-light parameters.
+
+"""
+
+import sys
+sys.path.insert(0, '../../sipmpdf')
+sys.path.insert(0, '../')
+
+import matplotlib.pyplot as plt
+import src.sipmanalyze.formats as forms
+import awkward as ak
+import numpy as np
+
+from scipy.signal import find_peaks
+import scipy.integrate as integrate
+from scipy.optimize import curve_fit
+
+
+def import_data(file_location,plot=True):
+  container = forms.waveform.waveform_container.from_root(file_location)
+  tint = container.settings.timeintervals
+  vadc = container.settings.adc_val
+  area = ak.sum(container.waveforms[:, 5:30], axis=-1) * tint * vadc * -1
+  amin, amax = ak.min(area), ak.max(area)
+  nbins = int((amax - amin) / tint / vadc / 4)
+
+  n, bins, patches= plt.hist(x=area, bins=nbins, range=(amin, amax), label='r')
+  total_elements=sum(n)
+  bin_width=(amax-amin)/nbins
+  
+  if plot:
```

Comment:
Try to split plot routines to a separate function (or maybe leave the plotting routines uniquely for the notebook). Functions in packages should have as little "side effects" (modifications to the program/memory space that persist after the function has be completed) as possible